frontend/packages/erd-core/src/features/gtm/utils/toolbarActionLogEvent.ts

```diff
@@ -0,0 +1,20 @@
+import { pushToDataLayer } from './pushToDataLayer'
+
+type ToolbarActionLogEvent = {
+  element: string
+  zoomLevel?: string
+  showMode?: string
+}
+
+export const toolbarActionLogEvent = ({
+  element,
+  zoomLevel,
+  showMode,
+}: ToolbarActionLogEvent) => {
+  pushToDataLayer({
+    event: 'click',
```

Comment:
I want to set the event in the toolbarActionLogEvent() function to 'toolbarAction'!

```suggestion
    event: 'toolbarAction',
```

Reply:
Oh, Sorry...🙇‍♀️

Fixed: https://github.com/liam-hq/liam/pull/326/commits/932ce8233165fb04e30405379d08ba7d8530eded

![ss 2572](https://github.com/user-attachments/assets/a6383705-6fe8-40c7-8608-fb296324955e)